dll/win32/profmap/profmap.spec

```diff
@@ -0,0 +1,5 @@
+@ stdcall -stub InitializeProfileMappingApi()
+@ stdcall RemapAndMoveUserA(str long str str)
+@ stdcall RemapAndMoveUserW(wstr long wstr wstr)
+@ stdcall -stub RemapUserProfileA()
```

Comment:
If you don't know the parameters, you want `@ stub FunctionName`.
(or since it's just one function and I looked it up... it's `([w]str long ptr ptr)`)

Reply:
Thanks. The stubs without `stdcall` work for me and still don't cause any Winlogon failures.
Btw, if it's not a secret, where you looked parameters of undocumented RemapUserProfileA and RemapUserProfileW functions? Maybe there was a syntax for them too? Then I'd able to add a stubs for these functions too.